### PR TITLE
[G7T5-39] Add load_env module and healthcheck endpoint

### PR DESCRIPTION
**Pull request by quinncheong**
# Description

Add module to load env vars from .env files.
Add healthcheck endpoint

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

No tests 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by theianchia — APPROVED** [review]
LGTMMMMMM 🚀

**Review by jonathantan1425 — CHANGES_REQUESTED** [review]

File: backend/app/core/config.py [hunk 1]
```diff
@@ -42,7 +45,7 @@ def assemble_cors_origins(cls, v: Union[str, List[str]]) -> Union[List[str], str
     #         return None
     #     return v
 
-    DB_SERVER: str = os.getenv("DB_SERVER")
+    DB_SERVER: str = os.environ.get("DB_SERVER")
```
[the remark this answers is not in the document]
Comment (quinncheong) [reply backend/app/core/config.py:48]: Good catch thanks

- [ ] Change to os.getenv

File: backend/scripts/test.sh [hunk 2]
```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (quinncheong) [reply backend/scripts/test.sh:1]: Ok I knew this was abit fishy when I saw a bunch of other files being deleted also. Let me revert

- [ ] Revert deleted files

**Review by quinncheong — COMMENTED** [review]
Added in fixes

**Pull request merged**